src/subgraph/orders/index.ts

```diff
@@ -11,44 +11,68 @@ import { mapOrdersArrayToInterface, mapSingleOrderToInterface } from '../common/
 import { EMPTY_BYTES32, getUniqueValuesFromArray } from '../../common';
 
 // Get all order by a user address
-export const getAllOrdersByUserAddress = async (chainId: Chain, userAddress: string, count: number = 10) => {
-  const subgraphEndpoint = getSubgraphEndpoint(chainId);
-
-  const subgraphResponse = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count));
-  return mapOrdersArrayToInterface(subgraphResponse);
+export const getAllOrdersByUserAddress = async (
+  chainId: Chain,
+  userAddress: string,
+  count: number = 10,
+): Promise<Order[]> => {
+  try {
+    const subgraphEndpoint = getSubgraphEndpoint(chainId);
+    const subgraphResponse = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count));
+    const orders = mapOrdersArrayToInterface(subgraphResponse);
+    if (!orders) throw new Error('Orders not found');
+    return orders;
+  } catch (error) {
+    throw error;
```

Comment:
We need to show 404 Http status code when this type of error occurs. Please create custom NotFoundError class and set status to 404 there.
refer - https://github.com/Parifi/parifi-microservices/blob/develop/src/error/resource-exists.error.ts